browser/js/content.js

```diff
@@ -0,0 +1,115 @@
+"use strict";
+
+function init() {
+  // Only do work on reddit.
+  if (!location.hostname.endsWith('.reddit.com')) return;
+
+  // FIXME: This can be more declarative in the future.
+  if (checkCompose.test(location.pathname)) injectCompose();
+  else if (checkProfile.test(location.pathname)) injectProfile();
+  else if (checkThread.test(location.pathname)) injectThread();
+}
+window.addEventListener('load', init);
+
+
+const checkCompose = /^\/message\/compose$/
+function injectCompose() {
+  // /message/compose
+  // TODO: ...
+  console.log("keybase: On compose.");
+}
+
+const checkProfile = /^\/user\//;
+function injectProfile() {
+  // /user/<user>
+  // TODO: ...
+  console.log("keybase: On profile.");
+}
+
+const checkThread = /^\/r\/\w+\/comments\/\w+\//;
+function injectThread() {
+  // /r/<subreddit>/comments/<id>/<slug>
+  console.log("keybase: On thread.");
+
+  for (let c of document.getElementsByClassName("comment")) {
+    const author = escape(c.getAttribute("data-author"));
```

Comment:
@shazow did you get a chance to think about this comment? I just want to make sure we know exactly what our escape functions are doing, especially since we're currently using a deprecated one.